DESCRIPTION

```diff
@@ -32,6 +32,7 @@ Imports:
     r2dii.plot,
     readr,
     readxl,
+    stringi,
```

Comment:
please remove this unneeded dep, will approve then

Reply:
done